srcpkgs/tokyocabinet/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'tokyocabinet'
+pkgname=tokyocabinet
+version=1.4.48
+revision=1
+build_style=gnu-configure
+makedepends="zlib-devel bzip2-devel"
+short_desc="Modern implementation of DBM"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="LGPL-2.1-only"
+homepage="https://fallabs.com/tokyocabinet/"
+distfiles="https://fallabs.com/tokyocabinet/tokyocabinet-"$version".tar.gz"
+checksum=a003f47c39a91e22d76bc4fe68b9b3de0f38851b160bbb1ca07a4f6441de1f90
+
+post_install() {
+	vlicense "$DESTDIR"/usr/share/tokyocabinet/COPYING
+	rm -f "$DESTDIR"/usr/share/tokyocabinet/COPYING
+	rm -r "$DESTDIR/usr/share/tokyocabinet/doc"
+}
+
+tokyocabinet-devel_package() {
+	depends="$makedepends tokyocabinet-${version}_${revision}"
+	short_desc="-development files"
+	pkg_install() {
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.a"
+		vmove usr/include
+		vmkdir usr/share/doc/tokyocabinet
```

Comment:
I'm not convinced of this and the doc/* move.

Reply:
I think docs make sense in the -devel package, are you sugesting to change the way they are copied/moved to the subpackage?

Reply:
There is nothing of value for development inside that doc folder.

There are a couple of images, an incomplete set of html and css files and a few pdf, ppt files.

Reply:
ΟΚ ;) . So, should I leave it in the main package or delete it from the destdir?